front/app/api/ideas/types.ts

```diff
@@ -18,7 +18,7 @@ export type IdeaReactingDisabledReason =
   | 'project_inactive'
   | 'not_ideation'
   | 'reacting_disabled'
-  | 'disliking_disabled'
+  | 'reacting_dislike_disabled'
```

Comment:
This was the main issue: the disabled reason was changed on the BE but not on the FE. This disabled reason was being used to hide the button